src/main/java/opinion/rules/impls/ShouldRule.java

```diff
@@ -0,0 +1,34 @@
+package opinion.rules.impls;
+
+import opinion.rules.SentenceRule;
+import opinion.sentences.Sentence;
+import opinion.sentences.Word;
+import proto.RuleResultOuterClass;
+
+
+/**
+ * Should rule -- if a sentence contains the word "should", it is an opinion
+ */
+public class ShouldRule extends SentenceRule {
```

Comment:
The naming of `ShouldRule` is a little ambiguous/confusing. Maybe consider naming it something more descriptive such as `ContainsShouldRule` (and update other e.g. `string`s appropriately).